test/unit/test_runner_config.py

```diff
@@ -300,6 +300,10 @@ def test_prepare():
     assert rc.env['AWX_ISOLATED_DATA_DIR'] == '/'
     assert rc.env['PYTHONPATH'] == '/:'
 
+    os.environ['PYTHONPATH'] = "/foo/bar"
+    rc.prepare()
+    assert rc.env['PYTHONPATH'] == "/foo/bar:/:"
```

Comment:
Is the trailing `:` required?

Reply:
it's not *not* required, and can be helpful during future concatenations.